client/js/lounge.js

```diff
@@ -17,7 +17,6 @@ $(function() {
 		"/notice",
 		"/op",
 		"/part",
-		"/query",
```

Comment:
Thanks @xPaw, my bad for forgetting that `/query` was merely an alias for `/whois`...